RiotAPI/CreateAndSendRequest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RitoForCustoms.RiotAPI
+{
+    class CreateAndSendRequest
+    {
+        public async static Task<string> SendRequest(Uri myUri, HttpClient httpclient, string valAPI = null, string keyAPI = null)
+        {
+            HttpRequestMessage request = new HttpRequestMessage();
+            request.RequestUri = myUri;
+            request.Method = HttpMethod.Get;
+            if (valAPI != null & keyAPI != null)
```

Comment:
'&' can be exchanged for '&&'. Second part of the condition won't be executed when the first is false.